vspace/src/main/webapp/WEB-INF/views/exhibition/module.html

```diff
@@ -131,10 +131,46 @@ <h3 class="textDiv">[[${module.name}]]</h3>
          
          <div style="position: relative;top: 6.5%;">
             <h3 style="text-align:center;">[[${currentSlideCon.name}]]</h3>
-            <div id="blocks" class="Group_8_Class" th:each="contents: ${currentSlideCon.contents}">
-                <img th:if="${contents['class'].simpleName ==  'ImageBlock'}" id="${contents.id}" class="imgDiv" th:src="@{'/api/image/'+${contents.image.id}}" style="max-width:57%">
-                <div th:if="${contents['class'].simpleName ==  'TextBlock'}" id="${contents.id}" class="textDiv">[(${contents.htmlRenderedText()})]</div>
-                <div th:if="${contents['class'].simpleName == 'ChoiceBlock'}" id="${contents.id}" class="textDiv" style="margin: 1%;">
+
+            <div th:id="blocks" class="Group_8_Class" th:each="contents: ${currentSlideCon.contents}">
+                <img th:if="${contents['class'].simpleName ==  'ImageBlock'}" th:id="${contents.id}" class="imgDiv" th:src="@{'/api/image/'+${contents.image.id}}">
+                <div th:if="${contents['class'].simpleName ==  'TextBlock'}" th:id="${contents.id}" class="textDiv">[(${contents.htmlRenderedText()})]</div>
+                <div th:if="${contents['class'].simpleName ==  'BiblioBlock'}" th:id="${contents.id}" class="biblioDiv">
+               
+                	<p class="list-group-item" >Bibliography<br>
+                	Title : <span>[(${contents.biblioTitle})]</span><br>
+                	 Description: <span>[(${contents.description})]</span></p>
+                	<ul class="list-group">
+	                	<div  th:each="contentRef: ${contents.references}">
+							<li class="list-group-item" style = "font-size: 15px" >
+								<div  th:id="${contentRef.id}" class="referenceDiv">
+								<p  class="col-md-10">Reference Type : [[${contentRef.type}]]</p>
+								<p  class="col-md-10">Reference Title : [[${contentRef.title} ]] </p>
+								<p  class="col-md-10">Reference Author : [[${contentRef.author}]]</p>
+								<p  class="col-md-10">Reference Year : [[${contentRef.year}]]</p>
+								<p  class="col-md-10">Reference Volume : [[${contentRef.volume}]]</p>
+								<p  class="col-md-10">Reference Issue : [[${contentRef.issue}]]</p>
+								<p  class="col-md-10">Reference pages : [[${contentRef.pages}]]</p>
+								<p  class="col-md-10">Reference Editors : [[${contentRef.editors}]]</p>
+								
+								<p  class="col-md-10">Reference Note : [[${contentRef.note}]]</p>
+								<a th:href='${contentRef.url}'><p  class="col-md-12">Reference URL :  [[${contentRef.url}]]</p></a>
+								
```

Comment:
without looking at the page, this looks wrong. It should show the normal reference formatting here.